inginious/frontend/plugins/task_editorial/pages/task_editorial.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from inginious.frontend.plugins.multilang.problems.languages import get_all_available_languages
+from inginious.frontend.pages.course_admin.task_edit import CourseEditTask
+
+_TASK_EDITORIAL_TEMPLATE_PATH = os.path.join(os.path.dirname(__file__), "templates")
+
+def check_task_access(task):
+
+    return task.get_accessible_time().is_open()
+
+def editorial_task_tab(course, taskid, task_data, template_helper):
+
+    tab_id = 'tab_editorial'
+    link = '<i class="fa fa-graduation-cap fa-fw"></i>&nbsp; ' + _("Task editorial")
+
+    task_environment = task_data.get('environment')
+
+    if(task_environment == "multiple_languages" or task_environment == "Data Science" or task_environment == "HDL" or task_environment is None):
```

Comment:
`task_environment in {"multiple_languages", "Data Science", "HDL"} or task_environment is None`